seed/challenges/02-javascript-algorithms-and-data-structures/basic-data-structures.json

```diff
@@ -394,18 +435,64 @@
       "translations": {}
     },
     {
+      "id": "587d7dac367417b2b25184d3",
+      "title": "Basic Data Structures: Introduction to Objects",
+      "description": [
+        [
+          "",
+          "",
+          "The next data structure we will discuss is the JavaScript <dfn>object</dfn>. Like arrays, objects are a very fundamental part of JavaScript. However, it is probably safe to say that objects surpass arrays in flexibility, usefuleness and in their overall importance to the language &mdash; in fact, you may have heard this line before: 'In JavaScript, everything is an object.'<br><br>While an understanding of objects is important to understand the inner workings of JavaScript functions or JavaScript's object-oriented capabilities, JavaScript objects at a basic level are actually just <dfn>key-value pair</dfn> stores, a commonly used data structure across almost all programming languages. Here, we will confine our discussion to JavaScript objects in this capacity.",
+          ""
+        ],
+        [
+          "",
+          "",
+          "Key-value pair data structures go by different names depending on the language and the specific details of the data structure. The terms <dfn>dictionary</dfn>, <dfn>map</dfn>, and <dfn>hash table</dfn> all refer to the notion of a data structure in which specific keys, or properties, are mapped to specific values.<br><br>Objects, and other similar key-value pair data structures, offer some very useful benefits. One clear benefit is that they allow us to structure our data in an intuitive way; properties can be nested to an arbitrary depth, and values can be anything, including arrays and even other objects.<br><br>Largely due to this flexibility, objects are also the foundation for <dfn>JavaScript Object Notation</dfn>, or <dfn>JSON</dfn>, which is a widely used method of sending data across the web.",
+          ""
+        ],
+        [
+          "",
+          "",
+          "Another powerful advantage of key-value pair data structures is <dfn>constant lookup time</dfn>. What this means, is that when you request the value of a specific property, you will get the value back in the same amount of time (theoretically) regardless of the number of entries in the object. If you had an object with 5 entries or one that held a collection of 1,000,000, you could still retrieve property values or check if a key exists in the same amount of time.<br><br>The reason for this fast lookup time, is that internally, the object is storing properties using some type of hashing mechanism which allows it to know exactly where it has stored different property values. If you want to learn more about this please take a look at the optional Advanced Data Structures challenges. All you should remember for now is that <strong>performant access to flexibly structured data make key-value stores very attractive data structures useful in a wide variety of settings</strong>.",
```

Comment:
Change "the object is storing properties using some type of hashing mechanism..." to "the object is storing properties using a hashing mechanism...". Otherwise, it sounds unsure.